webui/react/src/pages/F_ExpList/F_ExperimentList.tsx

```diff
@@ -159,11 +175,18 @@ const F_ExperimentList: React.FC<Props> = ({ project }) => {
     }
   }, [settings.filterset, isLoadingSettings]);
 
-  const [selectedExperimentIds, setSelectedExperimentIds] = useState<number[]>([]);
-  const [excludedExperimentIds, setExcludedExperimentIds] = useState<Set<number>>(
-    new Set<number>(),
-  );
-  const [selectAll, setSelectAll] = useState(false);
+  const [selectedExperimentIds, setSelectedExperimentIds] = useState<number[]>(() => {
+    if (isLoadingSettings) {
+      return [];
+    }
+    return settings.selectedExperiments;
+  });
+  const [excludedExperimentIds, setExcludedExperimentIds] = useState<Set<number>>(() => {
+    if (isLoadingSettings) {
+      return new Set();
+    }
+    return new Set(settings.excludedExperiments);
+  });
```

Comment:
question: is there a reason selected should be an array but excluded should be a set?

Reply:
I was keeping them the same type as before, but yes it does seem strange for them to be different. If I were making a single change, I'd make excluded into an array because we're storing it that way in settings

Reply:
both values being arrays makes sense to me